docs/running/notebooks/02_distributed_workloads.ipynb

```diff
@@ -196,21 +170,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "{'results': [{'0': 0.25, '1': 0.25, '2': 0.25, '3': 0.25},\n",
-       "  {'0': 0.4931987115505575,\n",
-       "   '1': 0.4931987115505575,\n",
-       "   '2': 0.0068012884494424,\n",
-       "   '3': 0.0068012884494424},\n",
-       "  {'0': 1.0}]}"
+       "{'results': [{'0': 1.0}, {'0': 1.0}, {'0': 1.0}]}"
```

Comment:
what caused the results to look so different?

Reply:
stochastic nature of quantum simulation